### PR TITLE
Fix cjs build exports

### PR DESCRIPTION
**Pull request by bradymadden97**
We wanted to keep the same API from v2, so `const Client = require('@replit/database')` still worked.

Right now it required `const Client = require('@replit/database').default;`

After updating the tests, before fixing the build:
```
 FAIL  test/__tests__/cjs.test.cjs > cjs smoke test
TypeError: Client is not a constructor
 ❯ test/__tests__/cjs.test.cjs:5:14
      3| 
      4| test('cjs smoke test', () => {
      5|   const db = new Client();
       |              ^
      6| 
      7|   expect(db).not.toBeUndefined()

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

 Test Files  1 failed | 2 passed (3)
      Tests  1 failed | 14 passed (15)
   Start at  19:19:26
   Duration  6.24s (transform 262ms, setup 2ms, collect 218ms, tests 5.18s, environment 1ms, prepare 839ms)
```

After updating the build to make `const Client = require('@replit/database')` work:

```
 RUN  v1.4.0 /home/runner/database-node-1

 ✓ test/__tests__/cjs.test.cjs (1)
 ✓ test/__tests__/mjs.test.mjs (1)
 ✓ test/__tests__/index.test.ts (13) 5194ms

 Test Files  3 passed (3)
      Tests  15 passed (15)
   Start at  19:22:03
   Duration  5.96s (transform 193ms, setup 0ms, collect 236ms, tests 5.21s, environment 1ms, prepare 531ms)
```

### CODE REVIEW

File: .replit [hunk 1]
```diff
@@ -1,6 +1,6 @@
 run = "npm run test"
 hidden = [".config", "package-lock.json"]
-modules = ["nodejs-20:v8-20230920-bd784b9"]
+modules = ["nodejs-with-prybar-18:v26-20240401-269b323"]
```
Comment (airportyh) [inline .replit:3]: Why did we downgrade to this module of all things? Prybar is deprecated.

File: package.json [hunk 2]
```diff
@@ -31,7 +29,8 @@
     "dist"
   ],
   "devDependencies": {
-    "tsup": "^8.0.2",
+    "@types/node": "^18.11.18",
+    "tsup": "^7.2.0",
```
Comment (airportyh) [inline package.json:33]: Ah we had to downgrade tsup.

**Review by airportyh — APPROVED** [review]
Thanks and I appreciate the added tests.

File: .replit (hunk 1 above)
Comment (bradymadden97) [reply .replit:3]: i was actually just trying to match the semaphore environment which is node 18, is there another node 18 environment i should use?

Comment (bradymadden97) [reply .replit]: fixed

**Pull request merged**